presto-manager-common/src/main/java/com/teradata/prestomanager/common/ApiRequester.java

```diff
@@ -111,37 +112,37 @@ public ApiRequester build()
 
         public Builder pathMethod(String method)
         {
-            uriBuilder.path(resource, method);
+            uriBuilder.path(resource, requireNonNull(method));
             return this;
         }
 
         public Builder resolveTemplate(String name, Object value)
         {
-            uriBuilder.resolveTemplate(name, value);
+            uriBuilder.resolveTemplate(requireNonNull(name), requireNonNull(value));
             return this;
         }
 
         public Builder queryParam(String name, Object... values)
         {
-            uriBuilder.queryParam(name, values);
+            uriBuilder.queryParam(requireNonNull(name), requireNonNull(values));
```

Comment:
You can have a query parameter without values.
It's not so useful in our current API, but it is a valid condition.